src/util/adb.js

```diff
@@ -418,3 +418,16 @@ export default class ADBUtils {
     });
   }
 }
+
+export async function listADBDevices(adbBin?: string): Promise<Array<string>> {
+  const adbClient = new ADBUtils({adbBin});
+  return adbClient.discoverDevices();
+}
+
+export async function listADBFirefoxAPKs(
+  deviceId: string,
+  adbBin?: string
+): Promise<Array<string>> {
+  const adbClient = new ADBUtils({adbBin});
+  return adbClient.discoverInstalledFirefoxAPKs(deviceId);
+}
```

Comment:
Let's add also 2 new unit tests in [test.adb.js](https://github.com/mozilla/web-ext/blob/master/tests/unit/test-util/test.adb.js) to cover their behavior (we may use sinon.stub to stub `ADBUtils.discoverDevices` and `adbclient.discoverInstalledFirefoxAPKs` to cover them with a unit test)

Reply:
Hi @rpl, I have added two new tests to [test.adb.js here](https://github.com/ankushduacodes/web-ext/blob/49f1e7175c3fb37b4fae7507afafe6bf108fb8f1/tests/unit/test-util/test.adb.js#L1314-L1340).

Please let me know if I am doing something wrong there.

Thank you !...